### PR TITLE
Improve theme support for Breadcrumb / Figure

### PR DESCRIPTION
**Pull request by ptbrowne**
- Use CSS variables for text color
- Breadcrumbs and Figure use those variables
- ThemeChooser component for easy switch between themes in styleguide
  (per component for now, would love to have it globally in the sidebar)

Preview: https://ptbrowne.github.io/cozy-ui/react/#!/Breadcrumbs

Edit: added the preview

### CODE REVIEW

File: docs/ThemeChooser.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react'
+import CozyTheme from '../react/CozyTheme'
+
+const ThemeChooser = ({ children }) => {
+  const [theme, setTheme] = useState('normal')
+  const handleChange = ev => setTheme(ev.target.value)
+  return (
+    <>
+      theme:{' '}
```
Comment (y-lohse) [inline docs/ThemeChooser.jsx:9]: Why do we need the `{' '}`?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cozy-ui",
-  "version": "35.14.0-beta.1",
+  "version": "35.18.0-beta.1",
```
Comment (y-lohse) [inline package.json:3]: Is this intended?

File: docs/ThemeChooser.jsx (hunk 1 above)
Comment (ptbrowne) [reply docs/ThemeChooser.jsx:9]: because there is a space after colons in English ? And it's the way to do this in JSX when a space precedes a component.

**Comment by ptbrowne** [comment]
Here is the preview yannick : https://ptbrowne.github.io/cozy-ui/react/#!/Breadcrumbs

File: package.json (hunk 2 above)
Comment (ptbrowne) [reply package.json:3]: No.

**Pull request merged**